cf-deployment.yml

```diff
@@ -698,7 +698,9 @@ instance_groups:
           cc-service-dashboards:
             authorities: clients.read,clients.write,clients.admin
             authorized-grant-types: client_credentials
-            scope: openid,cloud_controller_service_permissions.read
+            scopes:
+              - openid
```

Comment:
It looks like a lot of our other yaml lists don't indent the hyphen. It's purely semantic, but let's try and keep it consistent wherever possible :).

```
scopes:
- openid # prefer this
  - openid # over this
```